packages/shared/src/schemas.ts

```diff
@@ -78,6 +86,7 @@ export const openSwapDepositChannelSchema = z
     ccmMetadata: ccmMetadataSchema.optional(),
     maxBoostFeeBps: z.number().optional(),
     srcAddress: z.string().optional(),
+    refundParameters: refundParameters.optional(),
```

Comment:
is `refundParameters` the best name for this? its basically the parameters for fillOrKill right? so i would call it `fillOrKillOptions` so it sounds like the feature that it is.